website/sidebars.json

```diff
@@ -2,6 +2,7 @@
   "docs": {
     "Introduction": [
       "introduction/getting-started",
+      "introduction/quick-start",
```

Comment:
Personally not a fan of this placement. The first section out of this page is about installation, but the next page this navigates to is the Installation page?